playwright.config.js

```diff
@@ -0,0 +1,14 @@
+const config = {
```

Comment:
it is good to add more here, like I did in https://github.com/qlik-oss/sn-table/pull/988/files#diff-f679bf1e58e8dddfc6cff0fa37c8e755c8d2cfc9e6b5dc5520a5800beba59a92R3